### PR TITLE
Update purchasing.rst

### PR DESCRIPTION
**Pull request by greg1877**
Added BOM change requests.

### CODE REVIEW

File: content/pages/purchasing.rst [hunk 1]
```diff
@@ -157,6 +157,27 @@ exceed $500 per receipt.**
...
   staff, 2132 Bainer Office.
 
+Bill of Materials Change Orders
+===============================
+As professional engineers, you have a fiduciary responsibility to spend your sponsor’s 
+funds in the best interest of the project.  Therefore, it is critical that you plan your 
+purchases prudently.  That said, there is no way to foresee all the stumbling blocks 
+that will occur over the course of this quarter and additional purchases are almost certain.
+
+There will be a cap of two additional BOM changes for the remained of the quarter. Any
```
Comment (moorepants) [inline content/pages/purchasing.rst]: remainder

Comment (moorepants) [inline content/pages/purchasing.rst]:
```suggestion
There will be a cap of two additional BOM changes for the remainder of the quarter. Any
```

File: content/pages/purchasing.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ EME 185 A/B Capstone MAE Purchasing And Out-Of-Pocket Reimbursement Guidelines
 Key Guidelines
 ==============
 
-- Expenses must first be approved before purchased.
+- Expenses must first be approved before purchased.  There is a cap of two additional purchase requests.
```
Comment (moorepants) [inline content/pages/purchasing.rst]: Need to be careful about difference in BOM approvals and purchase requests.

If they are doing purchasing through OPS they can submit as many of them as they want.

If the TAs are doing purchasing through OPS then they can 1 one per two weeks.

Both of those are different than their 3 BOM submissions (orginal + 2 change orders).

**Comment by moorepants** [comment]
@greg1877 I left some comments, once addressed, and the merge button is green, you can merge.

**Comment by greg1877** [comment]
added information from 4/5/19 comments

**Pull request merged**